distribution/docs/src/main/resources/content/_configuring/exporting-existing-configurations-contents.adoc

```diff
@@ -32,7 +32,8 @@ image::exporting_configuration_step3.png[Exporting Step 3]
 
 To export the current ${branding} configuration from the ${command-console}:
 
-. Type in `migration:export <directory>`. This command creates the exported configuration file `exported-${project.version}.zip` which is saved to the specified directory. If no directory is specified it will default to `<${branding}_HOME>/exported`
+. Run the `migration:export` command from the ${command-console}.
+. A file named `exported-${project.version}.zip` will be created in the `exported` directory underneath `<${branding}_HOME>`, but an alternative location can be specified with the `migration:export`. Copy this file to a secure place.
```

Comment:
❗️  This whole section is the same as the export steps in the "Exporting/Importing Configuration Settings" section. That section gets placed right before this one, so it's very repetitive. One of them should be removed.

Reply:
Since this one still references the admin UI, I will leave it and address your comment in the other ticket where we will be removing the UI button.